revise FAQ to reflect 1.42 release terminology (#746)
deltachat/deltachat-pages: en/help.md

However, if both of you are in a green-checkmarked chat 
(see "Shared Chats" in the Contact profile) 
it's better to re-add them by letting them _scan the Group Invite QR code_
from the group member list. 
This will automatically resolve the warning for yourself 
and for all members of the green-checkmarked chat. 

**They sent a mail through a webmail interface or another e-mail app
and will get back to use Delta Chat soon again.**

Simply accept the warning.
As soon as your contact uses Delta Chat again, 
guaranteed end-to-end encryption will be automatically re-established. 

**They stopped using Delta Chat at all**

Accept the warning and remove the contact from any active green-checkmarked group 
(which you can find in "Shared chats" in the Contact profile). 

Don't fret too much about the "May not be end-to-end encrypted anymore" warning.
Sometimes remaining in contact is more important than end-to-end encryption.
Chances are that ["Transport Layer Encryption" (TLS)](#tls) still meaningfully protects 
the confidentiality of your messages between your device and the e-mail server. 

**Why don't other messengers like Signal show such a warning?**

> Signal does not provide a practical scheme to protect Signal users

Author: yes, i know but this is arguably not a "practical scheme" as discussed in the second intro paragraph of https://countermitm.readthedocs.io/en/latest/new.html#securing-communications-against-network-adversaries which is quoted in the next sentence.

> Signal does not provide a practical scheme to protect Signal users

Author: hum, but maybe the whole signal-note is misplaced here, i notice.

> Signal does not provide a practical scheme to protect Signal users

Author: k, i changed the paragraph.

### What does the green checkmark and "guaranteed end-to-end encryption" mean? {#e2eeguarantee}

Chat titles with green checkmarks
(<img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/green_checkmark.png" />)
mean that all messages in the chat will be end-to-end encrypted
and can not be read or altered by compromised e-mail servers or Internet providers. 
Joining green-checkmarked group chats
safely spreads everybody's encryption information (and green checkmarks)
in a manner that guarantees end-to-end encryption in the group and among members. 

Contact profiles with green checkmarks 
(<img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/green_checkmark.png" />)
mean that messaging a contact is currently guaranteed to be end-to-end encrypted.
Every green-checkmarked contact either did a direct [QR-scan](#howtoe2ee) with you
or was introduced by a another green-checkmarked contact.
Introductions happen automatically when adding members to groups. 
Whoever adds a contact to a green-checkmarked group becomes an introducer 
to those members who didn't yet know about the added contact.

> In a Contact profile you can tap on the "Introduced by ..." text repeatedly

Reviewer: ```suggestion
In a contact profile you can tap on the "Introduced by ..." text repeatedly
```

until you get to the one with whom you directly did a [QR-scan](#howtoe2ee).

> Note that in a Contact profile you may see and tap introducers 

Reviewer: ```suggestion
Note that in a contact profile you may see and tap introducers 
```

but there is no green checkmark in the profile title. 
This usually means that the contact ["sent a message from another device"](#nocryptanymore). 

For more in-depth discussion of "guaranteed end-to-end encryption"
please see [Secure-Join protocols](https://countermitm.readthedocs.io/en/latest/new.html)
and specifically read about "Verified Groups", the technical term
of what is called here "green-checkmarked" or "guaranteed end-to-end encrypted" chats.

### A contact "sent a message from another device", what can i do? {#nocryptanymore}

> "Messages may not be end-to-end encrypted anymore" means
> that your contact uses inconsistent or no end-to-end encryption at all.
> Here are the possible causes and respective mitigations.

Reviewer: We need to adapt this sentence to the new device message, too, e.g.:

```suggestion
Here are the possible causes and respective mitigations for getting warned
that a contact "sent a message from another device":
```

I think that `inconsistent or no end-to-end encryption at all` doesn't include meaningful information for most users, that's why I left it away in my suggestion.

**Your contact is using Delta Chat on a second device (phone or laptop)**

If they have another device with a Delta Chat app running,
they better remove the account from the new device 
and add it [as a second device through a QR code scan](#multiclient).
As soon as they message you afterwards, the warning will be gone
and guaranteed encryption is established with both devices of your contact. 

**Your contact reinstalled Delta Chat using their old account login**
 

> If they have [a backup file](#backup) 

Reviewer: ```suggestion
If they have [a backup file](#backup), 
```

  settings. You can set a timeframe between "after an hour" and "after a year";
  this way, *all* messages will be deleted from your device as soon as they are
  older than that.

## Encryption and Security

### Which standards are used for end-to-end encryption?

[Autocrypt](https://autocrypt.org) is used for automatically
establishing end-to-end encryption with contacts and group chats.
Autocrypt uses a limited and [secure subset of the OpenPGP standard](#openpgp-secure).
End-to-End encrypted messages are marked with a padlock 
<img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/padlock.png" alt="padlock"/>.

[Secure-Join protocols](https://countermitm.readthedocs.io/en/latest/new.html)
are used for establishing chats with guaranteed end-to-end encryption 
which protects against network attacks and compromised servers.
Chats marked with a green checkmark 
<img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/green_checkmark.png" alt="green checkmark"/> 
guarantee end-to-end encrypted messages. 

### How can i know if messages are end-to-end encrypted? {#whene2e}

> All end-to-end encrypted messages carry a padlock. 
> <img style="vertical-align:middle; width:10.5em; margin:1px" src="../assets/home/screenshot-with-padlock.png" alt="padlock"/>.

Reviewer: Right now there are two points.

```suggestion
All end-to-end encrypted messages carry a padlock 
<img style="vertical-align:middle; width:10.5em; margin:1px" src="../assets/home/screenshot-with-padlock.png" alt="padlock"/>.
```

or with a line break:

```suggestion
All end-to-end encrypted messages carry a padlock<br/>
<img style="vertical-align:middle; width:10.5em; margin:1px" src="../assets/home/screenshot-with-padlock.png" alt="padlock"/>.
```

> End-to-end encryption is guaranteed if there is a green checkmark next to the chat title. 
> <img style="vertical-align:middle; width:10.5em; margin:1px" src="../assets/home/screenshot-with-greencheckmark.png" alt="padlock"/>.  

Reviewer: Same here:

```suggestion
End-to-end encryption is guaranteed if there is a green checkmark next to the chat title: 
<img style="vertical-align:middle; width:10.5em; margin:1px" src="../assets/home/screenshot-with-greencheckmark.png" alt="padlock"/>.  
```

or 

```suggestion
End-to-end encryption is guaranteed if there is a green checkmark next to the chat title:<br/>
<img style="vertical-align:middle; width:10.5em; margin:1px" src="../assets/home/screenshot-with-greencheckmark.png" alt="padlock"/>.  
```

Reviewer 2: i fixed that with recent commit

  settings. You can set a timeframe between "after an hour" and "after a year";
  this way, *all* messages will be deleted from your device as soon as they are
  older than that.

## Encryption and Security

### Which standards are used for end-to-end encryption?

[Autocrypt](https://autocrypt.org) is used for automatically
establishing end-to-end encryption with contacts and group chats.
Autocrypt uses a limited and [secure subset of the OpenPGP standard](#openpgp-secure).
End-to-End encrypted messages are marked with a padlock 
<img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/padlock.png" alt="padlock"/>.

[Secure-Join protocols](https://countermitm.readthedocs.io/en/latest/new.html)
are used for establishing chats with guaranteed end-to-end encryption 
which protects against network attacks and compromised servers.
Chats marked with a green checkmark 
<img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/green_checkmark.png" alt="green checkmark"/> 
guarantee end-to-end encrypted messages. 

### How can i know if messages are end-to-end encrypted? {#whene2e}

All end-to-end encrypted messages carry a padlock

> <img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/padlock.png" alt="padlock"/>.

Reviewer 3: More screenshots more good :)

> <img style="vertical-align:middle; width:1.5em; margin:1px" src="../assets/home/padlock.png" alt="padlock"/>.

Author: i added this -- and also a screenshot for a green chat.

**End-to-end encryption is established automatically**

> after a first message was received from an Autocrypt-capable contact.
> Delta Chat defaults to using end-to-end encryption with that contact.

Reviewer 3: I like her suggestion. I think we could even let out the "default" sentence:

```suggestion
after a first message was received from a contact using Delta Chat
(or [another Autocrypt-capable client](https://autocrypt.org/dev-status.html)).
This means that the first message(s) are usually unencrypted, and
when the contact replies, all messages are encrypted.
```

> after a first message was received from an Autocrypt-capable contact.
> Delta Chat defaults to using end-to-end encryption with that contact.

Reviewer: Maybe writing "sometimes" or "often" instead of "usually" would prevent the surprise when the user does a securejoin and even the first message is encrypted:

```suggestion
after a first message was received from a contact using Delta Chat
(or [another Autocrypt-capable client](https://autocrypt.org/dev-status.html)).
This means that the first message(s) are sometimes unencrypted, and
when the contact replies, all messages are encrypted.
```

> after a first message was received from an Autocrypt-capable contact.
> Delta Chat defaults to using end-to-end encryption with that contact.

Reviewer 3: ```suggestion
after a first message was received from a contact using Delta Chat
(or [another Autocrypt-capable client](https://autocrypt.org/dev-status.html)).
This means that the first message(s) are sometimes unencrypted, and
as soon as the contact replies, all messages are encrypted.
```

Sounds good, but then we should change "when" to "as soon as" - the word "when" suddenly has a relation to "sometimes" which it did not have to "usually", because "when" and "sometimes" are both temporal. just a nitpick :D

> Delta Chat defaults to using end-to-end encryption with that contact.

Author: in this [commit](https://github.com/deltachat/deltachat-pages/pull/746/commits/388baf6568419ee27b8cbe8f44f4c674b4636ae4) , i stroke this explanation entirely as i think it's confusing/difficult to understand especially in conjunction with guaranteed e2ee.   The autocrypt-mode is not really the recommended default mode anymore, or shouldn't be described as such. (Pre-1.42 it was).  so we shouldn't first explain the somewhat tricky-to-explain autocrypt-mode. In the end, i think we need some drawings/videos explaining what happens.  Until then, anyone more interested can go and checkout https://autocrypt.org/level1.html

> Delta Chat defaults to using end-to-end encryption with that contact.

Reviewer 3: fine as well. it's a valid point that we can now tell everyone who wonders "but are the messages really encrypted?" that they can ensure that easily.

## Encryption and Security 

### Which standards are used for end-to-end encryption? 

[Autocrypt](https://autocrypt.org) is used for automatically
establishing E2E-encryption with contacts and group chats. 
Autocrypt uses a limited and [secure subset of the OpenPGP standard](#openpgp-secure). 

[Secure-Join protocols](https://countermitm.readthedocs.io/en/latest/new.html) 
are used for establishing chats with guaranteed end-to-end encryption ("green checkmark")
that protects against network attacks and compromised servers.

### When will messages be E2E-encrypted? 

Both individual or group chats carry a green checkmark 
if all communication in the chat is [guaranteed to be end-to-end encrypted](#howtoe2ee)

End-to-end encryption also takes place when all participants in a chat
use an Autocrypt-compliant e-mail app: 
After a first message was received from an Autocrypt-capable contact 

> Delta Chat defaults to using E2E-encryption with that contact (and vice versa). 

Reviewer: My user didn't know what "vice versa" means, maybe we should replace it with "the other way round" or just remove it - at least I don't understand what's supposed to also be the case the other way round here.

```suggestion
Delta Chat defaults to using E2E-encryption with that contact. 
```

Delta Chat also never was vulnerable to the "Direct Exfiltration" EFAIL attack
because it only decrypts `multipart/encrypted` messages
which contain exactly one encrypted and signed part,
as defined by the Autocrypt Level 1 specification.

### Is a message exposed in cleartext if end-to-end encryption is not available? {#tls}

Even if your messages are not guaranteed to be end-to-end encrypted, 
they are still protected from Internet providers like cell or cable companies. 
However, your and your recipient's e-mail providers 
may read, analyze or even modify your messages, 
including any attachments,
if they are not end-to-end encrypted. 

Delta Chat by default uses strict 

> [TLS encryption](https://en.wikipedia.org/wiki/Transport_Layer_Security)) 

Reviewer: ```suggestion
[TLS encryption](https://en.wikipedia.org/wiki/Transport_Layer_Security) 
```

and, more importantly, the sender and receiver addresses.
E-mail servers need receiver addresses to route and
deliver messages to recipient's devices.

### How to protect metadata and contacts when a device is seized? {#device-seizure}

Both for protecting against metadata-collecting e-mail servers
as well as against the threat of device seizure
we recommend to use a Delta Chat optimized [e-mail server instance](https://delta.chat/serverguide)
to create pseudonymous temporary accounts through QR-code scans.
Note that Delta Chat apps on all platforms support multiple accounts
so you can easily use action-specific "1-week" or "1-month" accounts next to your "main" account
with the knowledge that all temporary account data, along with all metadata, will be deleted.
Moreover, if a device is seized then contacts using temporary e-mail accounts
can not be identified easily, as compared to messengers which reveal
phone numbers in chat groups which in turn are often associated with passport identities.

### How can i check encryption information?

You may check the end-to-end encryption status manually in the "Encryption" dialog

> (user profile on Android/iOS or right-click a user's chat-list item on desktop).

Author: sorry this is just a white-space change, no content change (editor-shenenigans)

the connection is safe.

### How can I check the encryption status of messages?

A little **padlock** in a message bubble denotes
that the message was properly end-to-end encrypted from the given sender.
If there is **no padlock**, the message was not properly end-to-end encrypted
most likely because the sender uses an app or webmail interface
without support for end-to-end--encryption.

### Why do I see unencrypted messages?

If a contact uses a non-Autocrypt e-mail app,
all messages involving this contact (in a group or 1:1 chat)
will not be end-to-end encrypted, and thus not show a "padlock" with messages.
Note that even if your contacts use Delta Chat on their account,
they might also use a non-Autocrypt e-mail app on that account
which then may cause intermittently unencrypted messages.
Replying unencrypted to unencrypted messages is mandated by Autocrypt
to prevent unreadable messages on the side of your contacts
and their non-Autocrypt e-mail app.

### How can i get an end-to-end encrypted chat with a Delta Chat contact who sometimes uses webmail or another non-Autocrypt e-mail app? 

If you need a safely end-to-end encrypted chat with a contact 

> who is using their account both with Delta Chat and non-Autocrypt apps (e.g. webmail)

Reviewer: ```suggestion
who is using their account both with Delta Chat and non-Autocrypt apps (e.g. webmail),
```

it's best to setup [guaranteed end-to-end encryption with them](#howtoe2ee)
and then create a guaranteed end-to-end encrypted group chat with you two as members. 
In this group chat all messages will be end-to-end encrypted 
even if the direct chat between you two has a

> ["Messages may not be end-to-end encrypted anymore"](#nocryptanymore) warning. 

Reviewer: ```suggestion
["… sent a message from another device"](#nocryptanymore) warning. 
```

  try the **manual transfer** described below

> ### Manual Transfer {backup}

Reviewer 2: to make the link work, the following syntax is needed:
```suggestion
### Manual Transfer {#backup}
```
